packages/api/controllers/nomination.js

```diff
@@ -47,11 +47,14 @@ const createNomination = async (req, res) => {
     const { providerEmailAddress } = req.body;
     const newNomination = await db.Nomination.create(req.body);
     const nominations = await db.Nomination.findAll();
-    const hasProviderBeenValidated = nominations.some( (nom) => {
-      return nom.providerEmailAddress === providerEmailAddress && nom.emailValidated === true
-    })
-    if(!hasProviderBeenValidated) {
-      verifyHcEmail(newNomination.dataValues)
+    const hasProviderBeenValidated = nominations.some((nom) => {
+      return (
+        nom.providerEmailAddress === providerEmailAddress &&
+        nom.emailValidated === true
```

Comment:
Minor nitpick, but you don't need the `=== true` part here. Since it's a boolean, it already has a true or false value.